libexec/basher-_clone

```diff
@@ -5,18 +5,26 @@
 
 set -e
 
-if [ "$#" -ne 1 ]; then
+if [ "$#" -lt 1 ]; then
   basher-help _clone
   exit 1
 fi
 
-package="$1"
+package=$1
 
 if [ -z "$package" ]; then
   basher-help _clone
   exit 1
 fi
 
+site=${2:-github.com}
```

Comment:
Isn't that what's there?  If you mean without "site=", then that would try to run "github.com" as a command, not set it as the default value.